### ISSUE TITLE
Multiple styling issues

### ISSUE THREAD

**Issue by jasursadikov** [maximilionus/telegram-dark-shell#18]
Using telegram business, tags are not displayed properly. Here and there I see colors mismatching. I see the last update was quite a while, will you make any updates soon?

**Comment by maximilionus** [comment]
@jasursadikov The only reason for the lack of major updates is that everything was displaying correctly with my test cases.

Would be great to know the Telegram client you encountered this issue on, a step-by-step on how to reproduce it, and some screenshots so I can diagnose the problem.

**Comment by jasursadikov** [comment]
![Screenshot_20240317-230946.png](https://github.com/maximilionus/telegram-dark-shell/assets/14846427/2dd32380-0a9d-4af5-8b01-0d2213e49b04)

I found this part uncolored
When you speed up audio, the x1, x1.5, x2 are not colored properly, like vice versa.

**Comment by jasursadikov** [comment]
![image](https://github.com/maximilionus/telegram-dark-shell/assets/14846427/aa8df315-7113-491a-89ac-5a9cea5ff20d)
I found this on the web version. The colors for custom premium colors are blank.

**Comment by jasursadikov** [comment]
@maximilionus any updates? :D

**Comment by maximilionus** [comment]
Yep, thank you for reporting these issues. I will look into them when I have some free time for this project.

**Comment by elifian** [comment]
I fixed it. 
https://t.me/addtheme/dark_shell_2

**Comment by elifian** [comment]
![Screenshot_20250103-164048](https://github.com/user-attachments/assets/bfc3418b-0f85-4fba-958d-fe3e1dcaee36)
![Screenshot_20250103-164104](https://github.com/user-attachments/assets/c2aa5e05-c195-4099-862b-94cce81664da)

**Comment by elifian** [comment]
1. Fixed the colors of the avatars. 
2. Background of the player. 
3. The color of the emoji status has been changed to white. 
4. Fixed the send video button (it was completely white).

**Comment by maximilionus** [comment]
All the issues are fixed with #21. Big thanks to @elifian for making it possible.
